requirements.txt

```diff
@@ -1,111 +1,80 @@
-# Wheel 0.25+ needed to install certain packages on CPython 3.5+
-# like Pillow and psycopg2
-# See http://bitly.com/wheel-building-fails-CPython-35
-# Verified bug on Python 3.5.1
-wheel==0.29.0
-
 # Django
-Django>=1.11.18,<2.0
+#Django>=1.11.18,<2.0
+Django==2.2.12
```

Comment:
How about locking down only to minor version number (django~=2.2.0) so we won't miss security patches?
The commented out old requirement is no longer needed.